projects/tinkerbell/rufio/docker/linux/Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-glibc
+ARG BUILDER_IMAGE
+
+FROM $BUILDER_IMAGE as ipmitool-builder
+
+WORKDIR /newroot
+
+RUN set -x && \
+    clean_install "ipmitool openssl-libs ncurses-libs readline" && \
```

Comment:
I think this is pulling in systemd which is not necessary in these. Can you post the list of packages being included and use the typical pattern we use for systemd and other things that are clearly not needed in the final image.?

Reply:
Sure! I ran `ldd` as part of the build and saw ipmitool needed the following
```
ldd /newroot/usr/bin/ipmitool
linux-vdso.so.1 (0x00007fffff378000)
libm.so.6 => /lib64/libm.so.6 (0x00007f258cc99000)
libreadline.so.6 => /lib64/libreadline.so.6 (0x00007f258ca53000)
libncurses.so.6 => /lib64/libncurses.so.6 (0x00007f258c82c000)
libtinfo.so.6 => /lib64/libtinfo.so.6 (0x00007f258c601000)
libcrypto.so.10 => /lib64/libcrypto.so.10 (0x00007f258c1ab000)
libc.so.6 => /lib64/libc.so.6 (0x00007f258be00000)
/lib64/ld-linux-x86-64.so.2 (0x00007f258d341000)
libdl.so.2 => /lib64/libdl.so.2 (0x00007f258bbfc000)
libz.so.1 => /lib64/libz.so.1 (0x00007f258b9e7000)
```